### ISSUE TITLE
Fontemon Website not displaying correctly for me

### ISSUE THREAD

**Issue by technobulb** [mmulet/font-game-engine#6]
What am I doing wrong? when I type on the website it comes up in plain English... 
![Screenshot](https://user-images.githubusercontent.com/84107091/123738113-34e18b80-d872-11eb-842a-69838c94a693.png)

**Comment by mmulet** [comment]
What browser and what OS? Also make sure your browser's language is set to English (United States)

**Comment by technobulb** [comment]
Windows 10. Tried both edge and chrome browse. Language is set correctly.

**Comment by mmulet** [comment]
Hmm, try opening up the dev console: https://developer.chrome.com/docs/devtools/open/ 

and see if there are any errors in the console:
https://developer.chrome.com/docs/devtools/console/

Also look in the network tab to see if the font is being downloaded https://developer.chrome.com/docs/devtools/network/

Do you use any adblockers or other extensions? Those could be blocking it.

**Comment by technobulb** [comment]
I tried turning off adblocker. Here are the errors

fontemon.html#player:1 Unchecked runtime.lastError: The message port closed before a response was received.
fontemon.html#player:1 Error handling response: TypeError: Cannot read property 'loggedOut' of undefined
    at chrome-extension://kcnhkahnjcbndmmehfkdnkjomaanaooo/contentscript.js:167:137
fontemon.html#player:1 Failed to decode downloaded font: https://www.coderelay.io/assets/fontemon.otf
fontemon.html#player:1 OTS parsing error: invalid sfntVersion: 1008821359

**Comment by mmulet** [comment]
That's very strange: the sfntVersion is the "magic number" for otf ( the first 4 bytes of the file). Usually, for .otf files, it should be 'OTTO' which is the ACII code for 0x4f54 544f (or 1330926671 in decimal ). [docs](https://docs.microsoft.com/en-us/typography/opentype/otspec180/otff).
I opened the .otf with a a hex editor, and it does indeed have 'OTTO' as the first 4 bytes. 
So, my best lead is that fontemon.otf is somehow being corrupted, but I don't know why. 

The SHA-256 hash of fontemon.otf is `B224B586AA42C2DA3A47C8E2871A0549A8412C335CF93ADBE950B91EA7944071`

Can download fontemon.otf and send me the SHA-256 hash? If not, send the file to me in an email (anything at all ) @ coderelay.io and I'll compute the hash.

**Comment by mmulet** [comment]
Now that I think about it, if the file is corrupt, try clearing your cache and redownloading the file. On chrome:
1. Open up dev tools
2. Press and hold the refresh button (in your normal browser window, not dev tools)
3. Select "Empty cache and Hard reload"

If it still doesn't work after this, try opening up the console and see if the invalid sfntVersion matches 1008821359, the same as before.

**Comment by technobulb** [comment]
Heres what I get after the hard reload:

fontemon.html:1 Failed to decode downloaded font: https://www.coderelay.io/assets/fontemon.otf
fontemon.html:1 OTS parsing error: invalid sfntVersion: 1008821359

And here is the hash of my downloaded file:
9fa30993fbdd822ab8476ee992056f45d0abf0e31205291fa2cd142c8ec8300f

When I open the .otf in notepad OTTO is indeed the first 4 characters.
After installing the .otf, I still cannot view the font in Microsoft Word, rather when I type, it comes up blank (but I do see Microsoft's spelling suggestion popups, indicating that it is indeed, registering my keystrokes)

**Comment by mmulet** [comment]
If the hashes don't match, then they're not the same file, very strange. I don't know why that would be happening. 
Random guess: Are you on a corporate or university ISP that could be trying to sanitize the file?

MS word doesn't work reliably. Despite being an image editor, fontemon works well in GIMP https://www.gimp.org. I've also got it to work in Windows font settings. 

Other things to try: The OTS parser used in Chrome is a standalone program https://github.com/khaledhosny/ots and we can find more error information by running ots on your downloaded font.

**Comment by technobulb** [comment]
OK so now that you mention, I do have a sort of filter installed, that does redirect all the traffic through its servers. Didn't think would affect my downloads at all...
